### DISCUSSION TITLE
[Help] catppuccin options

### DISCUSSION THREAD

**Discussion by edshamis** [LazyVim/LazyVim#507] [Q&A] [answered]
Found this snippet to change split border, but don't know how to append it to catppuccin opts
```lua
      local cp_api = require("catppuccino.api.colors")
      local err, colors = cp_api.get_colors("neon_latte")
      if err.status then -- good
        catppuccino.remap({}, {
          VertSplit = { fg = colors.magenta },
        })
      end

```

**Comment by folke** [comment, marked as answer]
Just put that code in a `config` function for the catpuccin. For example:

`~/.config/nvim/lua/plugins/catpuccin.lua`

```lua
return {

  -- catppuccin
  {
    "catppuccin/nvim",
    name = "catppuccin",
   config = function()
      -- add the code here
   end
  },
}

```

You can add the code above in any file under your plugins. It doesn't have to be named catpuccin

**Reply by edshamis** [reply]
@folke Thanks!
Here's the final working snip for anyone seeking
```lua
  {
    "catppuccin/nvim",
    name = "catppuccin",
    config = function()
      Colors = require("catppuccin.palettes").get_palette("mocha")
      require("catppuccin").setup({
        custom_highlights = function(Colors)
          return {
            VertSplit = { fg = Colors.overlay0 },
          }
        end,
      })
    end,
  },
```

**Reply by celfred** [reply]
Hello,
Any idea why this doesn't work in my config ?
`

return {
  "catppuccin/nvim",
  lazy = false,
  name = "catppuccin",
  --  -- you can do it like this with a config function
  -- config = function()
  --     Colors = require("catppuccin.palettes").get_palette("mocha")
  -- require("catppuccin").setup({
  --       custom_highlights = function(Colors)
  --         return {
  --           WinSeparator = { fg = Colors.flamingo },
  --         }
  -- end,
  -- })
  -- end,
  --   -- or just use opts table
  opts = {
    flavour = "latte",
  },
  {
    "LazyVim/LazyVim",
    opts = {
      colorscheme = "catppuccin",
    },
  },
}
`
I've tried all sorts of things (and the commented out lines as suggested in the preceding answers), but I still have no lunck (either no color change or no latte flavour…). I feel like the complete customisation stays unread, but I have no idea why :disappointed:

**Reply by dpetka2001** [reply]
@celfred This should do it
```lua
return {
  {
    "LazyVim/LazyVim",
    opts = {
      colorscheme = "catppuccin",
    },
  },

  {
    "catppuccin",
    opts = {
      flavour = "latte",
    },
  },
}
```

**Reply by celfred** [reply]
Thanks a lot ! It works :)
I don't understand why all examples I've read suggest catppuccin/nvim. I thought the line name='catppuccin' was supposed to tell LazyVim the correct name…
